cva6/sim/cva6.yaml

```diff
@@ -50,11 +50,11 @@
   tool_path: SPIKE_PATH
   tb_path: TB_PATH
   precmd: >
-    make -C <path_var> verilate variant=<variant> user-define="+define+WT_CACHE"
+    make -C <path_var> verilate target=<target> user-define="+define+WT_CACHE"
   cmd: >
     <path_var>/work-ver/Variane_testharness +time_out=40000000 +debug_disable=1 <elf>
   postcmd: >
-    <tool_path>/spike-dasm < ./trace_rvfi_hart_00.dasm > log
+    grep "ffffffe0005e5cd4" ./trace_rvfi_hart_00.dasm
```

Comment:
I call these "magic numbers", but they work without explanation.  We should either not use them, or use variable names that have some meaning.  Also, it is likely this number must match another number in a separate file.   It would be useful if this file and the others that need this magic number to obtain its value from a common source.

Reply:
@MikeOpenHWGroup , This modification has been removed from the current PR. New #1039 PR contains it, with the explanation of "magic numbers"